group/assignment6/assignment6.py

```diff
@@ -0,0 +1,112 @@
+'''
+There is a parking lot with N spaces and N-1 cars in it. Below is 
+an algorithm to rearrange the cars in a given way. Only one car can be 
+moved at a time to the empty slot. Start state and end state are 
+represented as lists of car numbers, and 0 signifies empty slot. 
+Thus, a number of a lot in which a car is placed is an index of that car
+number in the list. For example, [1, 2, 0, 3] is the start state. 
+Car #1 is in the lot #0, car #2 is in the lot #1, etc.
+'''
+
+def move_to_lot(current_state, move_from, move_to):
+    '''
+    Move a car from one slot to another. As we are always moving a car to
+    an empty slot, the function basically swaps an empty and occupied slots.
+
+    :param current_state: order of cars in the present moment
+    :param move_from: int, number of a lot from which the car is moved
+    :param move_to: int, number of an empty slot to which we move the car
+    :returns: modified current state
+    '''
+    current_state[move_from], current_state[move_to] = current_state[move_to], \
+                                                       current_state[move_from]
+    return current_state
+
+def find_empty(current_state):
```

Comment:
It is really cool that we have a separate method for this action, I think it looks like OOP style or something like this, if we change the type of our states we will need to rewrite this method only and don't need to rewrite the whole code